File: javascript/stats.js
import axios from 'axios';

const baseUrl = 'https://api.mangadex.org';

const mangaID = '0301208d-258a-444a-8ef7-66e433d801b1';

const resp = await axios({
    method: 'GET',
    url: `${baseUrl}/statistics/manga/${mangaID}`
});

const { rating, follows } = resp.data.statistics[mangaID];

console.log(
    'Mean Rating:', rating.average, '\n' +
    'Bayesian Rating:', rating.bayesian, '\n' +
    'Follows:', follows
);

File: archive/python/stats.py
manga_id = "0301208d-258a-444a-8ef7-66e433d801b1"

import requests

base_url = "https://api.mangadex.org"

r = requests.get(f"{base_url}/statistics/manga/{manga_id}")

rating, follows, *others = r.json()["statistics"][manga_id].values()

print(
    f"Mean Rating: {rating}\n"
    + f"Follows: {follows}"
)